app-misc/khard/khard-0.17.0-r2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+inherit distutils-r1
+
+DESCRIPTION="Console CardDAV client"
+HOMEPAGE="https://github.com/scheibler/khard"
+
+LICENSE="GPL-3"
+SLOT="0"
+
+if [[ "${PV}" == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/scheibler/khard"
+else
+	SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+	KEYWORDS="amd64 arm arm64 x86"
+fi
+
+IUSE=doc
+
+RDEPEND="
+	dev-python/atomicwrites[${PYTHON_USEDEP}]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/ruamel-yaml[${PYTHON_USEDEP}]
+	dev-python/ruamel-yaml-clib[${PYTHON_USEDEP}]
+	dev-python/unidecode[${PYTHON_USEDEP}]
+	dev-python/vobject[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	test? (
+		dev-python/setuptools_scm[${PYTHON_USEDEP}]
+	)
+	doc? (
+		dev-python/sphinx
+		dev-python/sphinx-autoapi[python]
+		dev-python/sphinx-autodoc-typehints
+	)
+"
+
+DOCS=( CHANGES CONTRIBUTING.rst README.md doc/source/examples/khard.conf.example )
+
+distutils_enable_tests setup.py
+
+src_compile() {
```

Comment:
This now shadows distutils-r1_src_compile, you need to call it first.

distutils-r1 exports a src_compile but you're overriding the implementation without calling the one it provides